ddl/serial_test.go

```diff
@@ -1288,7 +1288,6 @@ func (s *testSerialSuite) TestInvisibleIndex(c *C) {
 	tk.MustExec("insert into t values (1, 2)")
 	tk.MustQuery("select * from t").Check(testkit.Rows("1 2"))
 	// 2. Drop invisible index
-	tk.MustGetErrMsg("alter table t drop column a", "[ddl:8200]can't drop column a with index covered now")
```

Comment:
Please modify the case to test dropping a column that is covered by the `invisible index`, thanks!

Reply:
What about add drop column with invisible index on `db_intergration_test.go`?

Reply:
That would be fine.